astro-docs/src/components/markdoc/IndexPageCards.astro

```diff
@@ -7,7 +7,10 @@ export interface Props {
   path: string;
 }
 
-const { path } = Astro.props;
+const { path: rawPath } = Astro.props;
+// make sure paths are normalized to url slugs 
+// i.e. features/CI Features -> features/ci-features
+const path = rawPath.replace(/\s/, '-').toLowerCase();
```

Comment:
Do we need to strip non-alphanumeric characters like `&`, or is that already not in the path?

Reply:
Oh wait, I got confused, it's out own path. Ignore.